### DISCUSSION TITLE
An abnormal problem occurs in the query of billion-level data

### DISCUSSION THREAD

**Discussion by wutachiang** [milvus-io/milvus#18253] [Q&A and General discussion]
In the experiment of 100 million sift feature storage, there is a problem that the index cannot be loaded, and the attu display is consistent in loading. After milvus_cli is connected to the service, the 'list collections' displays the abnormal information 'message=ta
Testing at 1 million and 10 million features is normal.

![2022-07-13_09-16](https://user-images.githubusercontent.com/13926708/178629995-07cb1691-9fe9-46b7-9fdd-a464412aa4e4.png)
![2022-07-13_09-38](https://user-images.githubusercontent.com/13926708/178630014-3420b45d-7e78-4112-9812-85f2ec07eb6c.png)
![2022-07-13_09-18](https://user-images.githubusercontent.com/13926708/178630039-8d990ab9-bfcc-476d-bf50-be797bd2f4f9.png)

kubectl checks the container status and finds that the querynode has a large number of Evicted phenomena, and other pods are normal. Use the dashboard to view the cluster resources and find that the CPU and memory usage is not high.
Configuration: milvus 2.0.2, the cluster consists of 10 12-core 48G memory 100G SSD machines, and the pod resources are shown in the following figure
![2022-07-13_09-41](https://user-images.githubusercontent.com/13926708/178630323-73e1d957-b2ce-4b88-bdbb-185e95d1d8c2.png)

**Comment by yhmo** [comment]
Some of the query nodes restart 20~38 times, might be OOM issue. You can create an issue in **issues** panel, and provide log of the abnormal query nodes. It's probably a known issue of v2.0.2.
